models/Profile.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+
+const ProfileSchema = new mongoose.Schema({
+
+    name:{
+        type: String,
+      
+    },
+
+    bio:{
+        type: String,
+     
+    },
+     
+    birthday:{
+        type: Date,
+    }
+})
+
+const Profile = mongoose.model('Profile' , ProfileSchema)
+module.exports = Profile;
```

Comment:
Don't you already have a user schema?

Why create another model with more user properties?